src/main/java/com/spotify/github/v3/clients/RepositoryClient.java

```diff
@@ -546,15 +546,28 @@ public CompletableFuture<Branch> getBranch(final String branch) {
   }
 
   /**
-   * Get a specific branch.
+   * List some branches in repository
    *
-   * @return list of all branches in repository
+   * @return list of 30 branches in repository
+   * @deprecated Doesn't return all branches. Use {@link RepositoryClient#listAllBranches} instead
    */
+  @Deprecated
```

Comment:
Hi and thank you for your contribution! I don't think we really need to deprecate this method?
Did a similar PR recently: https://github.com/spotify/github-java-client/pull/176 without deprecating the non paged method. Otherwise I think this PR looks good!

Reply:
Somehow I think developers should be warned to not use the old method and point them in a better direction. Depreciation was the clearest way I could think of for doing that. But I agree that depreciation is a bit wrong in the sense that the old method isn't likely to be removed.

Reply:
The good think with deprecation is that it gives a visible warning in peoples IDEs. But OK, how about this?